CONTRIBUTING.md

```diff
@@ -7,31 +7,97 @@
 > This warning will be removed once we have settled on a reasonable set of
 > guidelines for contributions.
 
-Fork, then clone the repo:
+### 1 Fork the RedHatGov.io repo
 
-    git clone git@github.com:your-username/redhatgov.github.io.git
-    cd redhatgov.github.io
-    git remote add upstream https://github.com/RedHatGov/redhatgov.github.io
+Forking RedHatGov.io is a simple two-step process.
 
-    # Never push to upstream master
-    git remote set-url --push upstream no_push
+1. On GitHub, navigate to the https://github.com/RedHatGov/redhatgov.github.io
+repo.
+2. In the top-right corner of the page, click **Fork**.
 
-    # Confirm that your remotes make sense:
-    git remote -v
+That's it! Now, you have a [fork][git-fork] of the original RedHatGov/redhatgov.github.io
+repo.
 
-Get your local docs up to date:
+### 2 Create a local clone of your fork
 
-    git fetch upstream
-    git checkout docs
-    git rebase upstream/docs
+Right now, you have a fork of the RedHatGov.io repo, but you don't have the files in that repo on your computer. Let's create a [clone][git-clone] of your fork locally on your computer.
 
-Make your change.
+```sh
+git clone git@github.com:your-username/redhatgov.github.io.git
+cd redhatgov.github.io
 
-Push to your fork and [submit a pull request][pr].
+# Configure git to sync your fork with the original repo
+git remote add upstream https://github.com/RedHatGov/redhatgov.github.io
+
+# Never push to upstream repo
+git remote set-url --push upstream no_push
+```
+
+### 3 Verify your [remotes][git-remotes]
+
+To verify the new upstream repo you've specified for your fork, type
+`git remote -v`. You should see the URL for your fork as `origin`, and the URL for the original repo as `upstream`.
+
+```sh
+origin  git@github.com:your-username/redhatgov.github.io.git (fetch)
+origin  git@github.com:your-username/redhatgov.github.io.git (push)
+upstream        https://github.com/RedHatGov/redhatgov.github.io (fetch)
+upstream        no_push (push)
+```
+
+### 4 Modify your `docs`
+
+Get your local `docs` [branch][git-branch], up to date:
+
+```sh
+git fetch upstream
+git checkout docs
+git rebase upstream/docs
```

Comment:
@dudash I mimicked the [Kubernetes](https://github.com/kubernetes/community/blob/master/contributors/devel/development.md#workflow) documentation on contributing as a point of reference.

But, their docs could be dated by newer git functionality? I'm OK with either.